src/locale/locales/ca/messages.po

```diff
@@ -8594,15 +8594,15 @@ msgstr "Videojocs"
 
 #: src/view/com/util/post-embeds/VideoEmbed.web.tsx:163
 msgid "Video not found."
-msgstr ""
+msgstr "No s'ha trobat el vídeo"
```

Comment:
```suggestion
msgstr "No s'ha trobat el vídeo."
```
The only very tiny thing that I – a non-Catalan speaker, of course 😅 – spotted is a missing `.` at the end here

Reply:
Why I keep missing them haunts me. Can you tell me what program/utility do you use to spot the errors?
Thanks again for your useful contributions

Reply:
It's only a very tiny thing, I wasn't even sure if it was worth pointing out tbh 😅

I don't use a program, just my eyes 👀🤷🏻‍♂️